src/main/java/witchinggadgets/client/ClientEventHandler.java

```diff
@@ -229,24 +227,17 @@ public void renderGameOverlay(RenderGameOverlayEvent.Pre event) {
                         GL11.glDisable(GL11.GL_LIGHTING);
                         GL11.glDepthMask(false);
                         ClientUtilities.bindTexture("witchinggadgets:textures/models/white.png");
-                        GL11.glEnable(3042);
                         GL11.glEnable(GL11.GL_BLEND);
-                        GL11.glBlendFunc(770, 771);
+                        GL11.glEnable(GL11.GL_BLEND);
```

Comment:
Double enable of GL Blend?

Reply:
Yeah idk it was like that before but applyDecompilerCleanupToMain has revealed it.